tests/torch/features/test_sequential.py

```diff
@@ -208,3 +208,46 @@ def test_sequential_and_non_sequential_tabular_features(yoochoose_schema, torch_
     outputs = tab_module(torch_yoochoose_like)
 
     assert list(outputs.shape) == [100, 20, 203]
+
+
+def test_sequential_tabular_features_with_text_module(yoochoose_schema, torch_yoochoose_like):
+    from typing import List
+
+    import torch
+
+    from transformers4rec.torch.features.base import InputBlock
+
+    class NormalizeTextembeddings(InputBlock):
+        def __init__(self, features: List[str], **kwargs):
+            """
+            Parameters:
+                features: Names of text features
```

Comment:
does that mean the column name of the text input feature?

Reply:
yes, I updated the description.

Reply:
does that have to be text features? or we can give a more generalized description like `name of the external features we have pre-trained embeddings for`